tools/lsp/main.rs

```diff
@@ -253,6 +246,7 @@ fn main() {
                     env!("CARGO_PKG_VERSION_MINOR"),
                     env!("CARGO_PKG_VERSION_PATCH")
                 );
+                let _ = writeln!(file, "Command line arguments: {command_line}",);
```

Comment:
This will break the parsing done for the crash reporter: the second line is expected to be the location.

And we should be careful about sending the command line args can contain data that we might not want to sent for telemetry.